app/controllers/api/articles_controller.rb

```diff
@@ -3,4 +3,11 @@ def index
     articles = Article.all
     render json: articles, each_serializer: ArticlesIndexSerializer
   end
+
+  def show
+    article = Article.find(params[:id])
+    render json: article, serializer: ArticlesShowSerializer
+  rescue ActiveRecord::RecordNotFound => e
+    render json: { message: 'Something went wrong...' }, status: 422
```

Comment:
You could probably have a bit more descriptive message, like 'Something went wrong, this article was not found' for example 🧐  also, look up the status code for not found